Sync/BookmarkPayload.swift

```diff
@@ -352,8 +352,8 @@ open class BookmarkPayload: BookmarkBasePayload {
     }()
 
     lazy var tagsString: String = {
-        if self["tags"].type == Type.array {
-            return self["tags"].stringValue
+        if self["tags"].isArray() {
+            return self["tags"].stringValue() ?? "[]"
```

Comment:
Does this suffer from the same issue as `isString()`? Do we need to fix `isArray()` in a similar way?